api/src/opentrons/protocols/geometry/deck.py

```diff
@@ -42,7 +42,7 @@ class CalibrationPosition:
 
 
 class Deck(UserDict):
-    def __init__(self, load_name: str = STANDARD_DECK) -> None:
+    def __init__(self, load_name: str = "") -> None:
```

Comment:
Can this be optional, instead of empty-string-as-falsey?

```suggestion
    def __init__(self, load_name: Optional[str] = None) -> None:
```